README.md

```diff
@@ -63,29 +63,29 @@ The latest pravega releases can be found on the [Github Release](https://github.
 
 ## Quick Start
 
-Read [Getting Started](documentation/src/docs/getting-started.md) page for more information, and also visit [sample-apps](https://github.com/pravega/pravega-samples) repo for more applications.
+Read [Getting Started](getting-started.md) page for more information, and also visit [sample-apps](https://github.com/pravega/pravega-samples) repo for more applications.
 
 ## Running Pravega
 
-Pravega can be installed locally or in a distributed environment. The installation and deployment of pravega is covered in the [Running Pravega](documentation/src/docs/deployment/deployment.md) guide.
+Pravega can be installed locally or in a distributed environment. The installation and deployment of pravega is covered in the [Running Pravega](deployment/deployment.md) guide.
 
 ## Support
 
-Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://codecommunity.slack.com/#pravega) if you need any help.
+Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://pravega-io.slack.com/) if you need any help.
 Open an issue if you found a bug on [Github
 Issues](https://github.com/pravega/pravega/issues)
 
 ## Documentation
 
 The Pravega documentation of is hosted on the website:
-<http://pravega.io/docs/latest> or in the [documentation](documentation/src/docs) directory of the source code.
+<http://pravega.io/docs/latest> or in the [documentation](.) directory of the source code.
```

Comment:
Are you sure that the `[documentation](.)` link is working? In your PR branch it seems broken.

Reply:
yes, fixed it.